pkg/core/native/native_neo.go

```diff
@@ -527,19 +505,22 @@ func (n *NEO) distributeGas(ic *interop.Context, acc *state.NEOBalance) (*big.In
 	if ic.Block == nil || ic.Block.Index == 0 || ic.Block.Index == acc.BalanceHeight {
 		return nil, nil
 	}
-	gen, err := n.calculateBonus(ic.DAO, acc.VoteTo, &acc.Balance, acc.BalanceHeight, ic.Block.Index)
+	gen, err := n.calculateBonus(ic, acc)
```

Comment:
Which allows to reuse `ic.Block.Index` here (it's a bit more correct, even though it technically is the same as `BlockHeight()+1`).

Reply:
Done